### PR TITLE
Feature/deliverable1

### PR DESCRIPTION
**Pull request by rafaelroguz**

### CODE REVIEW

File: src/components/CarouselItem/CarouselItem.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { BannerImage, Container, Title } from "./CarouselItem.styled";
+
+const CarouselItem = ({ alt, categoryUrl, imageUrl, name }) => (
+  <Container href={categoryUrl || "#"}>
+    <BannerImage alt={alt} src={imageUrl} />
+    <Title>{name}</Title>
+  </Container>
+);
+
+CarouselItem.propTypes = {
```
Comment (rafaelroguz) [inline src/components/CarouselItem/CarouselItem.jsx:12]: Just curious: could I extract this object into a separate file and assign PropTypes, so I can export it and use it kind like an TS `interface`? JSON files are mocking DB results, so I was hoping to "create a model interface" for the response, just like in TS, but not sure if that's a good idea/practice

Comment (jazzcelaya) [reply src/components/CarouselItem/CarouselItem.jsx:12]: Actually it is a pretty good practice, so please go ahead (please let me know when you do so I don't forget to pull yur changes)

**Comment by rafaelroguz** [comment]
Found the issue with the main container adding padding outside the box:
`box-sizing: border-box;`, so the `max-width`: 1200px` takes in account the padding when calculating the width

**Pull request merged**